modules/dnn/src/darknet/darknet_io.cpp

```diff
@@ -149,25 +169,47 @@ namespace cv {
                     net->layers.push_back(lp);
 
                     if (use_batch_normalize)
-                    {
-                        cv::dnn::LayerParams bn_param;
-
-                        bn_param.name = "BatchNorm-name";
-                        bn_param.type = "BatchNorm";
-                        bn_param.set<bool>("has_weight", true);
-                        bn_param.set<bool>("has_bias", true);
-                        bn_param.set<float>("eps", 1E-6);	// .000001f in Darknet Yolo
-
-                        darknet::LayerParameter lp;
-                        std::string layer_name = cv::format("bn_%d", layer_id);
-                        lp.layer_name = layer_name;
-                        lp.layer_type = bn_param.type;
-                        lp.layerParams = bn_param;
-                        lp.bottom_indexes.push_back(last_layer);
-                        last_layer = layer_name;
-                        net->layers.push_back(lp);
+                        setBatchNorm();
+
+                    layer_id++;
+                    fused_layer_names.push_back(last_layer);
```

Comment:
Looks like this is extra changes? Or not?

Reply:
No, I didn't do any change here.

Reply:
Oh, got it, sorry!